### ISSUE TITLE
Is it possible to write this project in PHP?

### ISSUE THREAD

**Issue by hasanizad** [stewartmcgown/uds#109]
I really appreciate this unique project.
I'm not very familiar with Python. It took me a few days to convert this code to PHP, but I still haven't succeeded.
Sending and receiving files to Google Drive with PHP is very easy. The issue I could not get to is whether the mimetype should be attached to the file by Google methods itself or the program does this and uses Google methods only to send and receive files?
In other words, is it possible to save the files on the computer and then upload them manually to Google Drive and in this case the storage is still zero?

**Comment by stewartmcgown** [comment]
Unlimited Google docs has been killed by Google. Of course you could write this in PHP, it just wouldn't be any use.

**Comment by hasanizadt** [comment]
but it work for me.
i use the program and send file to GD and it have no any size.
how kill it?

**Comment by stewartmcgown** [comment]
June 2021 will be the last date you can upload anything using UDS without it counting against your storage.

It does indeed still work and you are welcome to try and write it in PHP!
